src/test_l3_dsDisplay.c

```diff
@@ -0,0 +1,462 @@
+
+    dsError_t status   = dsERR_NONE;
+
+    UT_LOG_INFO("Calling dsDisplayInit()");
+
+    status = dsDisplayInit();
+
+    UT_LOG_INFO("Result dsDisplayInit() dsError_t=[%s]", 
+                UT_Control_GetMapString(errorMappingTable, status));
+
+    DS_ASSERT(status == dsERR_NONE);
+
+    UT_LOG_INFO("Out %s", __FUNCTION__);
+
+}
+
+/**
+ * @brief Retrieves the display handle based on user-selected video port.
+ *
+ * Allows the user to select a video port and retrieves the corresponding display handle.
+ *
+ * **Test Group ID:** 03@n
+ * **Test Case ID:** 002@n
+ *
+ * **Test Procedure:**
+ * Refer to Test specification documentation
+ * [ds-display_L3_Low-Level_TestSpecification.md](../../docs/pages/ds-display_L3_Low-Level_TestSpecification.md)
+ */
+void test_l3_dsDisplay_get_handle(void)
+{
+    gTestID = 2;
+    UT_LOG_INFO("In %s [%02d%03d]", __FUNCTION__, gTestGroup, gTestID);
+
+    dsError_t status = dsERR_NONE;
+    int32_t choice = -1;
+    uint8_t numInputPorts = 0;
+    dsVideoPortType_t port = dsVIDEOPORT_TYPE_MAX;
+
+    numInputPorts = UT_KVP_PROFILE_GET_UINT32("dsDisplay/Number_of_ports");
+
+    UT_LOG_MENU_INFO("----------------------------------------------------------");
+    UT_LOG_MENU_INFO(" \t  Supported Video Port are:");
+    UT_LOG_MENU_INFO("----------------------------------------------------------");
+    UT_LOG_MENU_INFO("\t#   %-30s%s","Video Port", "Index");
+    for (int32_t i = 0; i < numInputPorts; i++) {
+        UT_LOG_MENU_INFO("\t%d.  %-30s%-2d", i+1, UT_Control_GetMapString(videoPortTypeMappingTable, gDSVideoPortConfiguration[i].typeid), gDSVideoPortConfiguration[i].index);
+    }
+    UT_LOG_MENU_INFO("----------------------------------------------------------");
+    UT_LOG_MENU_INFO(" Select the Video Port:");
+    scanf("%d", &choice);
+    readAndDiscardRestOfLine(stdin);
+
+    if(choice < 1 || choice > numInputPorts) {
+        UT_LOG_ERROR("Invalid Port choice");
+        goto exit;
+    }
+
+    port = choice - 1;
+
+    UT_LOG_INFO("Calling dsGetDisplay(IN:type:[%s], IN:index:[%d], OUT:handle:[])", 
+                UT_Control_GetMapString(videoPortTypeMappingTable, gDSVideoPortConfiguration[port].typeid), 
+                gDSVideoPortConfiguration[port].index);
+
+    status = dsGetDisplay(gDSVideoPortConfiguration[port].typeid, gDSVideoPortConfiguration[port].index, &gDisplayHandle);
+
+    UT_LOG_INFO("Result dsGetDisplay(IN:type:[%s], IN:index:[%d], Handle:[0x%0X]) dsError_t=[%s]", 
+                UT_Control_GetMapString(videoPortTypeMappingTable, gDSVideoPortConfiguration[port].typeid),
+                gDSVideoPortConfiguration[port].index, 
+                gDisplayHandle,
+                UT_Control_GetMapString(errorMappingTable, status));
+
+    DS_ASSERT(status == dsERR_NONE);
+
+    UT_LOG_MENU_INFO("Calling dsRegisterDisplayEventCallback");
+
+    UT_LOG_INFO("Calling dsGetEDID(IN:handle:[0x%0X], IN:cb[0x%0X]) ", gDisplayHandle, displayEventCallback);
```

Comment:
Here you have not called dsGetEdid(), but mentioned in logging. Please check it.

Reply:
Done